core/capabilities/ccip/ocrimpls/contract_transmitter.go

```diff
@@ -211,7 +222,7 @@ func (c *commitTransmitter) Transmit(
 		return fmt.Errorf("failed to generate UUID: %w", err)
 	}
 	zero := big.NewInt(0)
-	if err := c.cw.SubmitTransaction(ctx, c.contractName, c.method, args, fmt.Sprintf("%s-%s-%s", c.contractName, c.offrampAddress, txID.String()), c.offrampAddress, &meta, zero); err != nil {
+	if err := c.cw.SubmitTransaction(ctx, contract, method, args, fmt.Sprintf("%s-%s-%s", contract, c.offrampAddress, txID.String()), c.offrampAddress, &meta, zero); err != nil {
```

Comment:
Can't comment on the line in this PR but there is still some evm-specific signature splitting above (evmutil.SplitSignature) that may not work for Solana.

Reply:
ah, damn. Thanks for pointing that out.

Reply:
On further investigation, `evmutil.SplitSignature` is just poorly named. It should be named `ecdsautil.SplitSignature`. It's unlikely that we'd use a different signature for OCR reports, so this is not something that would change now or in the future.